src/test/java/com/g4s8/ghman/data/PgUserITCase.java

```diff
@@ -30,17 +30,19 @@
  *  methods of {@link com.g4s8.ghman.data.PgUser} class. Do not forget about
  *  all possible exceptions.
  */
-final class PgUserITCase extends DatabaseITCase {
+@ExtendWith(DatabaseExtension.class)
+final class PgUserITCase {
 
     @Test
     void throwsExceptionIfUserIsNotFound() {
-        MatcherAssert.assertThat(
+        new Assertion<>(
+            "Unexpected exception while selecting not existing user",
             Assertions.assertThrows(
```

Comment:
@olenagerasimova with cactoos-matchers, we use here the `Throws` class to assert something about a thrown exception.